framework/src/base/Moose.C

```diff
@@ -146,6 +146,9 @@ addActionTypes(Syntax & syntax)
   registerMooseObjectTask("add_ic",                       InitialCondition,       false);
   appendMooseObjectTask  ("add_ic",                       ScalarInitialCondition);
 
+  registerMooseObjectTask("add_fv_ic",                    FVInitialCondition,     false);
+  appendMooseObjectTask  ("add_fv_ic",                    ScalarInitialCondition);
```

Comment:
I'm not quite sure what this does. Does this allow building initial conditions for scalar variables in the `FVICs` syntax?

Reply:
Ahhh great catch!